pkgs/development/tools/goa/default.nix

```diff
@@ -21,5 +21,6 @@ buildGoPackage rec {
     description = "A framework for building microservices in Go using a unique design-first approach";
     license = licenses.mit;
     maintainers = [ maintainers.rushmorem ];
+    broken = true;
```

Comment:
cc @rushmorem

Reply:
Goa seemed to build right now, is a just go110 required? https://hydra.nixos.org/build/80201530

Reply:
goa does indeed build with `buildGo110Package`. I recently had build issues with goa so I mistakenly assumed it was _still_ broken.